vm/systemSmartContracts/staking.go

```diff
@@ -42,6 +42,8 @@ type stakingSC struct {
 	enableStakingEpoch       uint32
 	stakeValue               *big.Int
 	flagStake                atomic.Flag
+	flagStakingV2            atomic.Flag
```

Comment:
maybe rename the flags/ It is hard to read for someone who does not know the story.

Reply:
auction.flagStake -> auction.flagEnableStaking
auction.flagStakingV2 -> auction.flagEnableTopUp

staking.flagStake -> staking.flagEnableStaking
staking.flagStakingV2 -> staking.flagSetOwner